sources/Inputs/include/Behaviours.h

```diff
@@ -122,5 +122,52 @@ struct ConverterInterface {
   boost::optional<bool> voltageRegulationOn;  ///< voltage regulation parameter, for VSC converters only
   std::shared_ptr<HvdcLine> hvdcLine;         ///< hvdc line this converter is contained into
 };
+
+/// @brief Static var compensator (SVarC) behaviour
+struct StaticVarCompensator {
+  using SVarCid = std::string;  ///< alias for static var compensator id
+
+  /**
+   * @brief Constructor
+   *
+   * @param id the id of the SVarC
+   * @param bMin the minimum susceptance value of the SVarC
+   * @param bMax the maximum susceptance value of the SVarC
+   * @param voltageSetPoint the voltage set point of the SVarC
+   * @param VNom the nominal voltage of the bus of the SVarC
+   * @param UMinActivation the low voltage activation threshold of the SVarC
+   * @param UMaxActivation the high voltage activation threshold of the SVarC
+   * @param USetPointMin the low voltage set point of the SVarC
+   * @param USetPointMax the high voltage set point of the SVarC
+   * @param b0 the initial b of the SVarC
```

Comment:
the initial susceptance value of the SVarC

Reply:
OK